utopia-remix/app/types.ts

```diff
@@ -31,3 +31,23 @@ export function userToCollaborator(user: UserDetails): Collaborator {
     avatar: user.picture,
   }
 }
+
+export interface Operation {
+  projectId: string
+  projectName: string
+  type: OperationType
+}
+
+export type OperationType = 'rename' | 'delete' | 'destroy' | 'restore'
+
+export function operationsEqual(a: Operation, b: Operation): boolean {
+  return a.projectId === b.projectId && a.type === b.type
```

Comment:
Should this include `projectName`? If not I'd be tempted to change what this function is called so it doesn't give people a false impression of what it checks.

Reply:
good point!

Reply:
[`7962936` (#4970)](https://github.com/concrete-utopia/utopia/pull/4970/commits/79629367287c8ba48ffa9ae21acac55b827c53f9)